README.md

```diff
@@ -1 +1,68 @@
-# Microverse-Hello-project
\ No newline at end of file
+# Microverse-Hello-project
+
+# Project Name
+-Microverse Hello
+> One paragraph statement about the project.
+
+![screenshot](./photo/screenshot.png)
```

Comment:
#### OPTIONAL

I'll mark this as a suggestion rather than a requirement but your preview screenshot is not working properly🤓 I assume there is a URL path issue that is causing this.